src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.cpp

```diff
@@ -116,18 +111,15 @@ void FlybyPoint::UpdateState(uint64_t CurrentSimNanos)
         }
 
         /*! compute inertial-to-reference DCM at time of read */
-        for (int i=0; i<3; i++) {
-            this->R0N[0][i] = ur_N[i];
-            this->R0N[1][i] = ut_N[i];
-            this->R0N[2][i] = uh_N[i];
-        }
+        this->R0N.row(0) = ur_N;
```

Comment:
:large_orange_diamond: Looks like this change snuck in here from he subsequent commit?

Reply:
Not sure I understood this one